pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -469,7 +469,9 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 		if isRemount {
 			verbosity = glog.Level(7)
 		}
-		glog.V(verbosity).Infof(volumeToMount.GenerateMsgDetailed("MountVolume.SetUp succeeded", ""))
+		simpleMsg, detailedMsg := volumeToMount.GenerateMsg("MountVolume.SetUp succeeded", "")
+		glog.V(verbosity).Infof(detailedMsg)
+		og.recorder.Eventf(volumeToMount.Pod, v1.EventTypeNormal, kevents.SuccessfulMountVolume, simpleMsg)
```

Comment:
Don't generate the event if `isRemount` is true (otherwise the event logs will be spammed).